ldk/go/uiClient.go

```diff
@@ -29,7 +29,11 @@ func (u *UIClient) ListenSearchbar(ctx context.Context, handler ListenSearchHand
 				break
 			}
 			if err != nil {
-				handler(resp.Text, err)
+				message := "Error with SearchbarStream"
```

Comment:
This is admittedly a nitpick. But I feel like error information shouldn't be returned in the message.
Thoughts on the following approach?
`
if err != nil {
  if resp == nil {
    handler("", err)
  } else {
    handler(resp.Text, err)
  }
  return
}
`

Reply:
I do think I prefer your way @BillyMurrayOlive.  Seems safer to have an empty message to encourage the callback to check the error.  Like the relevant details should all be in the `err` itself and not fragmented.  What if I just shorten it to the following?
```
if err != nil {
  handler("", err)
  return
}
```

Reply:
I think your abbreviated version is good. I was trying to accommodate returning resp.Text when you have it. But on second thought, when a function returns an error, you typically disregard the other arguments. So I don't think it is necessary.

Reply:
change applied, thank you Billy!